### ISSUE TITLE
How to train SMPL-based model?

### ISSUE THREAD

**Issue by Asakurayoho** [cure-lab/SmoothNet#15]
There are some problems in trying to reproduce the pw3d-spin-smpl experiment. 
* First, the key name in the `./data/poses/pw3d_spin_smpl/groundtruth` is misspelled, and the “pose” is written as “psoe”. 
* In addition, I tried to run the `./train_smoothnet.py` with default setting from pw3d_ spin_ 3d.yaml to train smpl-based model using oneeuro filter. It is found that although the train loss and acceleration errors have significantly decreased during the training stage, other metrics have become larger like MPJPE, MPVPE and PAMPJPE. I wonder if there is an error in my hyper-parameter setting. 
* Besides, does this repository consider adding the slerp filter function mentioned in the paper?

**Comment by juxuan27** [comment]
Hi, @Asakurayoho ! Thank you for your focus!
- We have corrected the spelling mistakes of `./data/poses/pw3d_spin_smpl/groundtruth` and thank you for pointing out the mistake.
- For SMPL-based methods, we recommend using checkpoints trained with 3D poses. For example, you can use pw3d_ spin_ 3d checkpoint for 3DPW-SPIN-SMPL testing.
- We will add the slerp filter into our repo as soon as possible.

**Comment by ailingzengzzz** [comment]
Hi @Asakurayoho,

We use the slerp filter from this file (https://github.com/hongsukchoi/TCMR_RELEASE/blob/master/evaluate.py#L216). The definition can be found here (https://github.com/hongsukchoi/TCMR_RELEASE/blob/ade5ab5e2bd8b1a26338406c0b272d2a16f85070/lib/utils/slerp_filter_utils.py#L1446)

**Comment by Asakurayoho** [comment]
Thanks a lot. I'll do more experiments on it.

**Comment by jihg88** [comment]
Hello author, I also found that based on smpl training, the loss can be reduced to a very low level, and the smoothing index acc also drops sharply, but in the end the index related to mpjpe increases instead. This is somewhat different from the training result of smpl-rot6d mentioned in the paper, which is slightly worse than that of 3D.
Is there anything that needs to be modified for smpl-based training, such as the loss function?
